api/v1beta1/service_types.go

```diff
@@ -234,12 +234,31 @@ func (a *Authorization) GetType() string {
 	return TypeUnknown
 }
 
+//ExternalRegistry specifies external source of data (i.e. OPA policy registry)
+type ExternalRegistry struct {
+	// Endpoint of the HTTP external registry.
+	// The endpoint must respond with either plain/text or application/json content-type.
```

Comment:
Just for the record, we say responses in `plain/text` or `application/json`, but only the latter is checked. This is fine and can be improved in the future if a third content-type (returned with a 200 OK status code) ever becomes problem when assumed to be a plain/text Rego policy.

Reply:
I agree. Should I add a comment with it or is it enough to leave it as a comment on the PR?